cmd/clusterctl/client/cluster/mover.go

```diff
@@ -676,7 +676,7 @@ func (o *objectMover) deleteSourceObject(nodeToDelete *node) error {
 }
 
 // checkTargetProviders checks that all the providers installed in the source cluster exists in the target cluster as well (with a version >= of the current version).
-func (o *objectMover) checkTargetProviders(namespace string, toInventory InventoryClient) error {
+func (o *objectMover) checkTargetProviders(toInventory InventoryClient) error {
```

Comment:
Removing watching namespace logic from this method is a leftover from https://github.com/kubernetes-sigs/cluster-api/pull/4666